run_dir/design/user_management.html

```diff
@@ -106,10 +110,14 @@ <h4><i class="fa fa-wrench mr-2"></i>User</h4>
             </div>
             <div class="row my-2">
               <div class="col mr-4">
-                <label for="pickUser" class="fw-bold">Modify Name</label>
+                <label for="pickUser" class="fw-bold">User Name</label>
                 <div>
                   <input class="form-control" id="formModifyUserName" placeholder="" disabled type="text">
                 </div>
+                <label for="modifyName" class="fw-bold">Name</label>
+                <input class="form-control" id="formModifyName" placeholder="Name..." type="text">
```

Comment:
Name comes from the GoogleUser currently and we probably shouldn't allow it to be modified from genstat. People can update their own google profiles to get names right.

Reply:
Ah, true. I'll have a look. We could probably use that instead.

Reply:
True, I didn't consider this, but starting to think about it, would it work to fetch other users´names?